projects/habitat_objectnav/config_utils.py

```diff
@@ -3,25 +3,30 @@
 
 import habitat.config.default
 import yaml
-from habitat.config.default import Config
+from omegaconf import DictConfig
+from habitat_baselines.config.default import get_config as get_habitat_config
+import os
 
+def get_config(path: str, opts: Optional[list] = None):
+    config = get_habitat_config(path)
+    return config, ""
 
-def get_config(path: str, opts: Optional[list] = None) -> Tuple[Config, str]:
+def get_config_old(path: str, opts: Optional[list] = None):
```

Comment:
@theophilegervet @exhaustin we have a problem where a lot of the new configs are in a different format. How should we handle this?

Reply:
ok sounds good!

Reply:
previous comment was chris not taosha